charts/dial/examples/gcp/complete/README.md

```diff
@@ -0,0 +1,159 @@
+- [DNS records](https://cloud.google.com/dns/docs/set-up-dns-records-domain-name) configured for Chat and Core
+- [Google-managed SSL certificates](https://cloud.google.com/kubernetes-engine/docs/how-to/managed-certs) issued for Chat and Core
+- [Google Storage bucket](https://cloud.google.com/storage/docs/buckets)
+- [Google MemoryStore RedisCluster](https://cloud.google.com/memorystore/docs/cluster)
+    - [Downloading the Certificate Authority](https://cloud.google.com/memorystore/docs/redis/manage-in-transit-encryption#downloading_the_certificate_authority)
+    - [Creating TrustStore](https://docs.oracle.com/cd/E19509-01/820-3503/ggfka/index.html)
+- [Google Identity](https://docs.epam-rail.com/Auth/Web/IDPs/google) as identity provider
+- [Google Vertex AI](https://cloud.google.com/vertex-ai/?hl=en) `gemini-1.5-pro` model deployed:
+  - [GCP Model Deployment Guide](https://docs.epam-rail.com/Deployment/Vertex%20Model%20Deployment)
+- [Azure OpenAI](https://learn.microsoft.com/en-us/azure/ai-services/openai/overview)
+  - [OpenAI Model Deployment Guide](https://docs.epam-rail.com/Deployment/OpenAI%20Model%20Deployment)
+- [Amazon Bedrock](https://docs.aws.amazon.com/bedrock/latest/userguide/what-is-bedrock.html) `anthropic.claude-v1` model deployed:
+  - [Bedrock Model Deployment Guide](https://docs.epam-rail.com/Deployment/Bedrock%20Model%20Deployment)
+
+## Expected Outcome
+
+By following the instructions in this guide, you will successfully install the AI DIAL system with configured connection to the Vertex AI, OpenAI, Bedrock APIs.\
+Please note that this guide represents a very basic deployment scenario, and **should never be used in production**.\
+Configuring authentication provider, encrypted secrets, model usage limits, Ingress allowlisting and other security measures are **out of scope** of this guide.
+
+## Install
+
+1. Create Kubernetes namespace, e.g. `dial`
+
+    **Command:**
+
+    ```sh
+    kubectl create namespace dial
+    ```
+
+    **Output:**
+
+    ```console
+    namespace/dial created
+    ```
+
+1. Add Helm chart repository
+
+    **Command:**
+
+    ```sh
+    helm repo add dial https://charts.epam-rail.com
+    ```
+
+    **Output:**
+
+    ```console
+    "dial" has been added to your repositories
+    ```
+
+1. Copy [values.yaml](values.yaml) file to your working directory and fill in missing values:
+    - Replace `%%NAMESPACE%%` with namespace created above, e.g. `dial`
+    - Replace `%%DOMAIN%%` with your domain name, e.g. `example.com`
+    - Replace `%%DIAL_API_KEY%%` with generated value (`pwgen -s -1 64`)
+    - Replace `%%CORE_ENCRYPT_SECRET%%` with generated value (`pwgen -s -1 32`)
+    - Replace `%%CORE_ENCRYPT_KEY%%` with generated value (`pwgen -s -1 32`)
+    - Replace `%%NEXTAUTH_SECRET%%` with generated value (`openssl rand -base64 64`)
+    - Replace `%%REDIS_PASSWORD%%` with generated value (`pwgen -s -1 32`)
```

Comment:
In case of the complete setup, I don't think that we need password for redis from the generated value

Reply:
Done